src/standalone/Soil/boundary_conditions.jl

```diff
@@ -418,21 +485,20 @@ function ClimaLand.boundary_flux(
 )::ClimaCore.Fields.Field
     FT = eltype(Δz)
     # Approximate K_bc ≈ K_c, ψ_bc ≈ ψ_c (center closest to the boundary)
-    p_len = Spaces.nlevels(axes(p.soil.K))
-    # We need to project the center values onto the face space.
-    K_c = Fields.Field(
-        Fields.field_values(Fields.level(p.soil.K, p_len)),
-        axes(Δz),
-    )
-    ψ_c = Fields.Field(
-        Fields.field_values(Fields.level(p.soil.ψ, p_len)),
-        axes(Δz),
-    )
+    surface_space = axes(Δz)
+    K_c = get_top_surface_field(p.soil.K, surface_space)
```

Comment:
should these be named `K_f` and `\psi_f` since they're actually on the face space? or `K_bc` and `\psi_bc` for consistency w the parameters?

Reply:
hmmm well we actually do want the center value of psi for the finite difference approx of the gradient (Y_bc - Y_c). but we need to put it on the right space in order to carry out the computation (because clima core wont let us extract a level of a center valued field and combine with a face valued field).

Ill add some documentation